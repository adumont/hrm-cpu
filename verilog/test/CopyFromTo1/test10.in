16 e5
37 39
23 92
09 10
06 69
08 e2
41 e4
51 2b
19 bf
59 c3
49 82
50 10
01 5c
35 8c
40 21
06 1e
40 ac
16 a1
05 9e
22 50
42 59
59 24
50 6b
40 4c
08 0c
48 71
15 59
26 6e
34 2e
49 f0
47 8d
23 b5
07 2b
57 3e
58 cf
12 3d
45 e7
42 4a
18 b0
08 d9
53 b1
46 34
17 b3
