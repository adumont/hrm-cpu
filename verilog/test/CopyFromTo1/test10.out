e5
39
92
10
69
e2
e4
2b
bf
c3
82
10
5c
8c
21
1e
ac
a1
9e
50
59
24
6b
4c
0c
71
59
6e
2e
f0
8d
b5
2b
3e
cf
3d
e7
4a
b0
d9
